### PR TITLE
Add EOL statement to README

### LINKED ISSUE #194: Continued Support Post-Deprecation: Introducing Swift-Cloudant-LTS
Hi friends,

I think this is a perfectly fine library and it is still entirely compatible with Cloudant (Couch DB) Instances hosted in IBM Cloud and elsewhere (Apache Couch DB). Perhaps it doesn't have some new APIs, but in any case, it's a lot of good work and I don't agree with throwing it in the trash.

Therefore we I will be maintaining it here-> https://github.com/GigabiteLabs/swift-cloudant-lts

Some new features included in our latest release but not within the source repository:

- Support for Couch DB instances hosted on a sub directory (example.com/couch vs couch.example.com)
- Some additional debug logs

The latest release passes 100% in testing when targeting an IBM Cloud hosted Couch DB instance (at time of release).

Since this library deprecates on Dec 31 2021, please open new issues or move existing ones to our fork.

To the maintainers: much appreciated for all of your work thus far, if you want to add that link above to the Readme, that would be much appreciated & kind to our fellow developers.

Thanks,
Dan

### PR DESCRIPTION
**Pull request by vmatyus**
## Checklist

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../blob/master/DCO1.1.txt)
- [ ] Added tests for code changes _or_ test/build only changes
- [ ] Updated the change log file (`CHANGES.md`|`CHANGELOG.md`) _or_ test/build only changes
- [x] Completed the PR template below:

## Description
Fixes #194 

Add EOL statement to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # DEPRECATED
 
-**This library is now deprecated and will be end-of-life on Dec 31 2021.**
+**This library is now deprecated and not supported by IBM.**
```
Comment (ricellis) [inline README.md:3]: I don't think we need to say IBM, it isn't supported by anyone in this repo.
```suggestion
**This library is end-of-life and no longer supported.**
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-The library remains supported until the end-of-life date,
-but will receive only _critical_ maintenance updates.
+However, GigabiteLabs dedicated to continue this work. 
+Please visit their forked version with your concern: https://github.com/GigabiteLabs/swift-cloudant-lts
```
Comment (ricellis) [inline README.md:6]: I think we should be more vague here, the `README` in the archived repo might long outlast any given fork and new forks may emerge.
```suggestion
Please see the list of [forks](https://github.com/cloudant/swift-cloudant/network/members) for continuing development efforts (e.g. #194).
```

**Review by ricellis — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vmatyusGitHub) [reply README.md]: Changed in 5e0f275

Comment (vmatyusGitHub) [reply README.md]: Also removed the first sentence: 698734b

**Review by mojito317 — APPROVED** [review]

**Review by ricellis — APPROVED** [review]
Thanks @vmatyusGitHub , please squash the commits and then I'll merge it.

**Comment by vmatyusGitHub** [comment]
Finished with the squashing.

**Pull request merged**